pipelines/fbc-builder/README.md

```diff
@@ -237,7 +238,7 @@ This pipeline is pushed as a Tekton bundle to [quay.io](https://quay.io/reposito
 ### run-opm-command-oci-ta:0.1 task results
 |name|description|used in params (taskname:taskrefversion:taskparam)
 |---|---|---|
-|SOURCE_ARTIFACT| The Trusted Artifact URI pointing to the artifact with the application source code with generated file-based catalog from catalog-template.yml.| prefetch-dependencies:0.2:SOURCE_ARTIFACT|
+|SOURCE_ARTIFACT| The Trusted Artifact URI pointing to the final artifact.| prefetch-dependencies:0.2:SOURCE_ARTIFACT|
```

Comment:
Just a nit. This docstring change reduces clarity for me. I suggest using the same as in the task readme "The Trusted Artifact URI pointing to the artifact with the application source code, now including the generated file-based catalog."